crates/solidity/testing/snapshots/cst_output/ConstructorDefinition/simple/generated/0.4.11-failure.yml

```diff
@@ -5,8 +5,13 @@ Source: >
 
 Errors: # 1 total
   - >
-    InvalidProductionVersion(
-        ConstructorDefinition,
-    )
+    Error: Expected end of file.
```

Comment:
if we are having a scanner/parser distinction, I suggest putting it in:

- `snapshots/cst_output/parser`
- `snapshots/cst_output/scanner`

instead of the current form:

- `snapshots/cst_output`
- `snapshots/scanner`

is this intended?

Reply:
I removed the scanner tests - they should use a different mechanism if we want to do that.